specification/azsadmin/resource-manager/network/Microsoft.Network.Admin/stable/2022-02-01/VirtualNetworkGatewayConnections.json

```diff
@@ -0,0 +1,275 @@
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network.Admin/adminVirtualNetworkGatewayConnections": {
+      "get": {
+        "x-ms-examples": {
+          "Returns a list of all virtual network gateway connections.": {
+            "$ref": "./examples/VirtualNetworkGatewayConnections/List.json"
+          }
+        },
+        "tags": [
+          "VirtualNetworkGatewayConnections"
+        ],
+        "operationId": "VirtualNetworkGatewayConnections_List",
+        "description": "Returns a list of all Virtual Network Gateway Connections.",
+        "parameters": [
+          {
+            "$ref": "../../preview/2015-06-15/Network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- The list of virtual network gateway connections has been returned.",
+            "schema": {
+              "$ref": "#/definitions/VirtualNetworkGatewayConnectionsList"
+            }
+          },
+          "default": {
+            "description": "Error Response.",
+            "schema": {
+              "$ref": "../../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    }
+  },
+  "definitions": {
+    "VirtualNetworkGatewayConnectionsList": {
+      "description": "List of Virtual Network Gateway Connections.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List of Virtual Network Gateway Connections.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/VirtualNetworkGatewayConnection"
+          },
+          "x-ms-identifiers": []
+        },
+        "nextLink": {
+          "description": "URI to the next page.",
+          "type": "string"
+        }
+      }
+    },
+    "VirtualNetworkGatewayConnection": {
+      "description": "The Virtual Network Gateway Connection.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "description": "Virtual Network Gateway Connection properties.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/VirtualNetworkGatewayConnectionProperties"
+        },
+        "location": {
+          "description": "Name of Azure Stack",
+          "type": "string",
+          "readOnly": true
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../preview/2015-06-15/Network.json#/definitions/Resource"
+        }
+      ]
+    },
+    "VirtualNetworkGatewayConnectionProperties": {
+      "description": "Virtual Network Gateway Connection properties.",
+      "type": "object",
+      "properties": {
+        "connectionState": {
+          "description": "Virtual Network Gateway Connection State.",
+          "$ref": "#/definitions/VirtualNetworkGatewayConnectionState"
+        },
+        "provisioningState": {
```

Comment:
>provisioningState [](http://example.com/codeflow?start=9&length=17)

Declare this as enum property specifying the allowed values.

Reply:
Addressed, thanks!